tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -420,10 +430,28 @@
 		 */
 		function updateLink() {
 			const $link = $( '#jetpack-beta-branch-link' );
-			const url = getLink();
+			const url = getLink()[ 0 ];
+			const query = getLink()[ 1 ];
```

Comment:
You're calling the function twice. Cheaper to do like
```suggestion
			const [ url, query ] = getLink();
```

Reply:
Much nicer.